examples/fastify-server/src/client/index.ts

```diff
@@ -50,13 +50,13 @@ async function start() {
   let randomNumberCount = 0;
 
   await new Promise<void>((resolve) => {
-    const unsub = anon.client.subscription('sub:randomNumber', null, {
+    const subscription = anon.client.subscription('sub:randomNumber', null, {
       next(data) {
         console.log('>>> anon:sub:randomNumber:received:', data);
         randomNumberCount++;
 
         if (randomNumberCount > 3) {
```

Comment:
I've noticed this pattern in more than one spot. Consider using `take(3)` instead. It's the same basic idea.